e2e-jar-test/pom.xml

```diff
@@ -29,7 +29,7 @@
       <dependency>
         <groupId>net.snowflake</groupId>
         <artifactId>snowflake-ingest-sdk</artifactId>
-        <version>2.3.0</version>
+        <version>3.0.0</version>
```

Comment:
i didn't hear back on the slack thread on this topic after friday so have been assuming its not a major version bump?